ModernWpf/Controls/Primitives/WindowHelper.cs

```diff
@@ -70,6 +71,25 @@ private static void OnUseModernWindowStyleChanged(DependencyObject d, Dependency
                 {
                     window.ClearValue(FrameworkElement.StyleProperty);
                 }
+
+                FixSizeToContent(window);
```

Comment:
It's possible to use `DefaultWindowStyle` directly without setting the `UseModernWindowStyle` property, so it may be a good idea to make `FixSizeToContent` an attached property, like the existing `FixMaximizedWindow`, to cover that case.

Reply:
Ah, good point. Done.

Reply:
Thanks.

Reply:
The workaround just runs once. After that, there's nothing to turn off. I think the attached property does not represent state, so we may be slightly abusing it.

I could switch to using a public method, but then the user can't apply the workaround in XAML or via a Style. They would need to use C#.

Do you have a preference? Thanks.

Reply:
In the latest commit I addressed it as you requested.